scripts/build.py

```diff
@@ -106,6 +165,17 @@ def ensure_directory_exists(filepath):
         os.makedirs(directory)
 
 
+def _ensure_files_exist(filepaths):
+    """Ensures if files exist.
```

Comment:
Ensures that files exist at the given filepaths.

Reply:
Done.